.github/workflows/main.yml

```diff
@@ -25,44 +25,54 @@ jobs:
         os: [ubuntu-latest]
         python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
         experimental: [false]
+        build: ['']
```

Comment:
OTAH, what do you think about using 'default'?

Reply:
Yeah, that could work. I've no strong opinion either way :)